tests/algorithms/image/threshold/test_local.py

```diff
@@ -227,8 +229,45 @@ def test_dispersion_extended_threshold(self):
         result4 = debug.final_mask()
         assert result2 == result4
 
+    def test_dispersion_electron_threshold(self):
+        from dials.algorithms.image.threshold import (
+            DispersionElectronThreshold,
+            DispersionElectronThresholdDebug,
+        )
+        from dials.array_family import flex
```

Comment:
this clobbers the scitbx import. Do you need both? Probably better to float this and add `as dials_flex` or something

Reply:
Removed the flex import. The test seems to pass the same anyway